sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/EnumForSealedEnumTypeSpec.kt

```diff
@@ -26,7 +26,7 @@ internal data class EnumForSealedEnumTypeSpec(
 ) {
     private val typeSpecBuilder = TypeSpec.enumBuilder(sealedClass.createEnumForSealedEnumName(enumPrefix))
         .addOriginatingElement(sealedClassCompanionObjectElement)
-        .addKdoc("An isomorphic enum for the sealed class [$sealedClass]")
+        .addKdoc("An isomorphic enum for the sealed class [%T]", sealedClass)
```

Comment:
Is this a Kotlin Poet thing?

Reply:
And the result of this is just to simplify the fully-qualified nature of the name right?

Reply:
Yep this is a Kotlin Poet thing, having the `%T` lets Kotlin Poet that `sealedClass` is a type, which may need to be imported. In this case, since we're always generating in the same package as the `sealedClass`, we don't need to do any importing and we can avoid fully-qualifying the name.

Reply:
Interesting OK. Kotlin Poet magic....

Reply:
✨ 

In this file, I ended up having to use the fully-qualified name for each of the objects in the enum constructors, since the enum value names match the objects. So it's not always perfect either.

Reply:
Gotcha.